include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_esimd_radix_sort.h

```diff
@@ -90,30 +90,28 @@ radix_sort(_ExecutionPolicy&& __exec, _Range&& __rng)
 
 namespace oneapi::dpl::experimental::esimd
 {
-template <typename _ExecutionPolicy, typename _Range, bool IsAscending = true, std::uint16_t WorkGroupSize = 256,
-          std::uint16_t ItemsPerWorkItem = 16, std::uint32_t RadixBits = 8>
+template <std::uint16_t WorkGroupSize, std::uint16_t DataPerWorkItem, bool IsAscending = true,
+          std::uint32_t RadixBits = 8, typename _ExecutionPolicy, typename _Range>
 void
 radix_sort(_ExecutionPolicy&& __exec, _Range&& __rng)
 {
     if(__rng.size() < 2)
         return;
-    oneapi::dpl::experimental::esimd::impl::radix_sort<
-        _ExecutionPolicy, _Range, IsAscending, WorkGroupSize, ItemsPerWorkItem, RadixBits>(
-            ::std::forward<_ExecutionPolicy>(__exec), __rng);
+    oneapi::dpl::experimental::esimd::impl::radix_sort<WorkGroupSize, DataPerWorkItem, IsAscending, RadixBits>
+        (::std::forward<_ExecutionPolicy>(__exec), __rng);
 }
 
-template <typename _ExecutionPolicy, typename _Iterator, bool IsAscending = true, std::uint16_t WorkGroupSize = 256,
-          std::uint16_t ItemsPerWorkItem = 16, std::uint32_t RadixBits = 8>
+template <std::uint16_t WorkGroupSize, std::uint16_t DataPerWorkItem, bool IsAscending = true,
```

Comment:
What do you think about swapping `bool IsAscending` and `std::uint32_t RadixBits`? All the numeric parameters will be grouped.

Reply:
Nevermind, the users will more likely change soring order rather than the number of bits.